flipper-plugin-realm/src/utils/Renderer.tsx

```diff
@@ -12,53 +13,59 @@ type TypeDescription = {
 export const renderValue = (
   value: unknown,
   property: TypeDescription,
-  schemas: SchemaObject[],
+  schemas: CanonicalObjectSchema[],
   inner?: boolean,
 ) => {
   if (value === null) {
-    return inner ? "null" : <Typography.Text disabled>null</Typography.Text>;
+    return inner ? 'null' : <Typography.Text disabled>null</Typography.Text>;
   }
   if (value === undefined) {
-    return inner ? "undefined" : <Typography.Text disabled>undefined</Typography.Text>;
+    return inner ? (
+      'undefined'
+    ) : (
+      <Typography.Text disabled>undefined</Typography.Text>
+    );
   }
   let schema;
   let returnValue: JSX.Element | string | number = '';
 
   switch (property.type) {
     case 'string':
+      //@ts-expect-error These type errors are okay because the Realm data types guarantee type safety here.
```

Comment:
I still wonder if we can just type the return instead to remove all these `ts-expect-error` cases.